src/draw.c

```diff
@@ -745,16 +749,16 @@ static void render_content(cairo_t *c, struct colored_layout *cl, int width, dou
 
                 /* Draw progress bar
                 * TODO: Modify draw_rounded_rect to fix blurry lines due to fractional scaling
-                * Note: the bar could be drawn a bit smaller, because the frame is drawn on top 
+                * Note: the bar could be drawn a bit smaller, because the frame is drawn on top
                 */
                 // left side (fill)
                 cairo_set_source_rgba(c, cl->highlight.r, cl->highlight.g, cl->highlight.b, cl->highlight.a);
-                draw_rounded_rect(c, x_bar_1, frame_y, progress_width_1, progress_height, 
+                draw_rounded_rect(c, x_bar_1, frame_y, progress_width_1, progress_height,
```

Comment:
Why is there a change in progress bar drawing?

Reply:
I just noticed that my editor removed some space 😅 ^^